app/javascript/mastodon/components/status_content.js

```diff
@@ -170,10 +170,6 @@ export default class StatusContent extends React.PureComponent {
   render () {
     const { status } = this.props;
 
-    if (status.get('content').length === 0) {
```

Comment:
If there is a CW but no text, why don't we move the CW to the text? Avoid annoying click-through. That's what we do when somebody writes a CW but no text locally.

Reply:
I have no strong feeling about this, except that if we do that, it might be better if it were strictly a display thing, so that the CW carries over on reply.

Reply:
I'm not personally a fan of the CW-carry-over behaviour at all... 99% of the time I've seen it the reply does not contain anything CW-worthy so it just makes you click extra for nothing. Like if the original post talks about a medical situation under a CW that's fair but then all the "Get well soon!" also get a CW that's really unnecessary.

Reply:
eh. changed that so a CW is considered as the toot's content if that content was empty, and replying here would lose the CW carry-over behavior, even though I don't really like that